### PR TITLE
add color for selected value as selectedValueColor

### PR DESCRIPTION
**Pull request by AliRn76**
I think the color of selected value would be needed most of time.

### CODE REVIEW

**Comment by paxcodes** [comment]
I suggest completely passing in a TextStyle for the `selectedValue`, that way there is complete flexibility of changing the color, fontSize, fontFamily, etc.

And the default style as well.

**Comment by AliRn76** [comment]
> I suggest completely passing in a TextStyle for the `selectedValue`, that way there is complete flexibility of changing the color, fontSize, fontFamily, etc.
> 
> And the default style as well.

That would be good , but he wont merge it anyway :)

**Comment by MarcinusX** [comment]
Hey, can we have 2 params in here? One for `textStyle` and other one for `selectedTextStyle`? :)
I will merge it then :)

**Comment by AliRn76** [comment]
Done  :)
I just set the `textStyle` and `selectedTextStyle` for them

File: lib/numberpicker.dart [hunk 1]
```diff
@@ -257,10 +269,6 @@ class NumberPicker extends StatelessWidget {
   }
 
   Widget _integerListView(ThemeData themeData) {
-    TextStyle defaultStyle = themeData.textTheme.body1;
```
Comment (MarcinusX) [inline lib/numberpicker.dart:260]: We should still use those styles if user didn't provide any styles.
I believe it go like that:
```
TextStyle defaultStyle = textStyle ?? themeData.textTheme.body1;
TextStyle selectedStyle =
       selectedTextStyle ?? themeData.textTheme.headline.copyWith(color: themeData.accentColor);
```
Same in other ocurrences

Comment (MarcinusX) [reply lib/numberpicker.dart:260]: Also, could you undo all new formatting? :) So that in one PR we can only see those changes that are important.

Comment (AliRn76) [reply lib/numberpicker.dart:260]: sry for that format problem :\ 
I didn't change them , this ide decide to change them :(
btw I fix them

Comment (Miamoto-Musashi) [reply lib/numberpicker.dart:260]: any news on this?

Comment (AliRn76) [reply lib/numberpicker.dart:260]: It's complete, we waiting for accepting the pull request :)

**Comment by paxcodes** [comment]
Thanks for merging @MarcinusX ! 🎉

**Pull request merged**